### ISSUE TITLE
unknown operation (80)

### ISSUE THREAD

**Issue by WayneBishop1** [libimobiledevice/idevicerestore#360]
IOS 14 Beta 5 developer wont install it keeps giving the following error.

unknown operation (80)

**Comment by demhademha** [comment]
Could you please provide the log with the -d flag??

**Comment by nikias** [comment]
@WayneBishop1 yes please provide debug output. `unknown operation` isn't an error by itself, it just means that idevicerestore doesn't have a descriptive text for this status operation which the device reports back.
